### PR TITLE
API Update deprecations

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10531

### CODE REVIEW

**Review by ChloeSartorelli — CHANGES_REQUESTED** [review]
Version numbers are incorrect - use `1` major version numbering instead. This will also need to be applied to any other deprecation notices in this module.

File: _legacy/GraphQL/CreateFileMutationCreator.php [hunk 1]
```diff
@@ -15,7 +16,7 @@
 }
 /**
  * @todo Allow file upload (https://github.com/silverstripe/silverstripe-graphql/issues/19)
- * @deprecated 4.8..5.0 Use silverstripe/graphql:^4 functionality.
+ * @deprecated 1.8.0 Use _graphql directory functionality instead
```
Comment (ChloeSartorelli) [inline _legacy/GraphQL/CreateFileMutationCreator.php]: The original message was more correct - the thing to use is the latest graphql, the directory is irrelevant.
The same message will need to be changed in a couple of other classes below as well.

**Review by ChloeSartorelli — CHANGES_REQUESTED** [review]

File: _legacy/GraphQL/CreateFileMutationCreator.php (hunk 1 above)
Comment (emteknetnz) [reply _legacy/GraphQL/CreateFileMutationCreator.php]: Updated

**Review by ChloeSartorelli — APPROVED** [review]
MOG

**Pull request merged**